test/mailers/batch_mailer_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class BatchMailerTest < ActionMailer::TestCase
+  test 'sends emails for MARC batch exports' do
+    ClimateControl.modify DISABLE_ALL_EMAIL: 'false' do
+      theses = [theses(:one), theses(:two)]
+      email = BatchMailer.marc_batch_email('marc.zip', 'fake zip file contents', theses)
+
+      # Send the email, then test that it got queued
+      assert_emails 1 do
+        email.deliver_now
+      end
+
+      # Make sure it was sent to the right person with the expected attachment.
+      assert_equal ['test@example.com'], email.from
+      assert_equal ['test-metadata@example.com'], email.to
+      assert_equal 'ETD MARC batch export', email.subject
+      assert_equal 'marc.zip', email.attachments.first.filename
```

Comment:
I think this tests the attachment filename effectively enough to remove the test from `MarcExportJobTest` that you had a comment on being kinda weird maybe?

Reply:
It's a bit different since we're not actually calling `MarcExportJob#filename` here.